app/src/main/java/com/github/se/polyfit/model/user/User.kt

```diff
@@ -64,7 +74,8 @@ data class User(
             familyName = map["familyName"] as String,
             givenName = map["givenName"] as String,
             email = map["email"] as String,
-            photoURL = Uri.parse(map["photoURL"] as String))
+            photoURL =
+                if (map["photoURL"] != "null") Uri.parse(map["photoURL"] as String) else null)
```

Comment:
Oh sure! Thnks

Reply:
Actually, since we have already set the default value to `null` in the user class, returning null is better than returning `Uri.Empty = ""`. I think we should have initially set up it to be `Uri.EMPTY` as default value. Anyway, now, it would cause a much bigger refactor so i think it would be best to leave it as null in the sake of time before m2